### ISSUE TITLE
Incorrectly reported line numbers

### ISSUE THREAD

**Issue by Jmeyering** [styled-components/stylelint-config-styled-components#19]
I've been having major issues with line numbers being reported incorrectly given the following component

```js
  import PropTypes from 'prop-types';
  import styled from 'styled-components';

  import { SUPPORTED_REVIEW_SITES } from '../../../constants';

  /* eslint-disable import/no-dynamic-require, prefer-template */
  const ReviewSiteLogo = styled.img.attrs({
    'data-pin-nopin': true,
    alt: ({ reviewSite }) => reviewSite,
>>  src: ({ reviewSite, icon }) => {
      if (icon) {
        return require('assets/images/reviewSites/' + reviewSite + '-icon.svg');
      }
      return require('assets/images/reviewSites/' + reviewSite + '-logo.svg');
    },
  })`
    display: block;

  `;
  /* eslint-enable import/no-dynamic-require, prefer-template */

  ReviewSiteLogo.defaultProps = {
    icon: false,
  };

  ReviewSiteLogo.propTypes = {
    /**
     * If passed, will render the icon form rather than the logo form.
     */
    icon: PropTypes.bool,

    /**
     * The name of the review site to render a logo or icon for
     */
    reviewSite: PropTypes.oneOf(SUPPORTED_REVIEW_SITES).isRequired,
  };

  /**
   * @component
   */
  export default ReviewSiteLogo;
```

I get 

```
src/components/atoms/ReviewSiteLogo/ReviewSiteLogo.jsx
 10:1  ✖  Unexpected empty line before closing backtick   block-closing-brace-empty-line-before
```

Of course the real error is on line 18.

**Comment by Jmeyering** [comment]
Another example

```js
    1 import React from 'react';
    2 import PropTypes from 'prop-types';
    3 import styled, { css } from 'styled-components';
    4 import { RichText } from 'horseman-components';
    5
    6 const StyledContent = styled.section`
    7   font-weight: 300;
    8
    9   color: ${({ inverse, theme }) => inverse ? 'white' : theme.colors.bodyGray};
   10   font-size: ${({ introText }) => introText ? '1.5em' : '1.25em'};
   11   line-height: ${({ introText }) => introText ? '2em' : '1.7em'};
   12
   13   ${props => props.centered && css`
   14     text-align: center;
   15   `}
   16
   17   a {
>> 18     color: ${({ theme }) => theme.colors.lightBlue};
   19     font-weight: bold;
   20     transition: .3s color ease-in;
>> 21     text-decoration: none;
   22
   23     &:hover, &:focus, &:active{
   24       color: ${({ theme }) => theme.colors.lightBlueHover};
   25     }
   26   }
   27
   28   strong {
   29     font-weight: 600;
>> 30   }
   31
   32   ul, ol {
   33     margin: 0;
   34     padding: 0 0 0 1em;
   35
   36     & > li {
   37       line-height: 1.75em;
   38     }
   39   }
   40 `;
   41
   42 const Content = ({ children, inverse, centered, introText }) =>
   43 (
   44   <StyledContent inverse={inverse} centered={centered} introText={introText}>
   45     <RichText html={children} />
   46   </StyledContent>
   47 );
   48
   49 Content.propTypes = {
   50   children: PropTypes.node.isRequired,
   51
   52   /**
   53    * Increases font-size and line-height (used mostly on top page sections)
   54    */
   55   introText: PropTypes.bool,
   56
   57   /**
   58    * Turns text white
   59    */
   60   inverse: PropTypes.bool,
   61
   62   /**
   63    * Centers content
   64    */
   65   centered: PropTypes.bool,
   66 };
   67
   68 /**
   69  * @component
   70  */
   71 export default Content;
```

And the output

```
 18:17  ✖  Expected a leading zero                                                 number-leading-zero
 21:12  ✖  Expected newline after ","                                              selector-list-comma-newline-after
 21:21  ✖  Expected newline after ","                                              selector-list-comma-newline-after
 21:30  ✖  Expected single space before "{" (block-opening-backtick-space-before)  block-opening-brace-space-before
 30:5   ✖  Expected newline after ","                                              selector-list-comma-newline-after
```

Looks like everything is shifted by ~ 2 lines

**Comment by mxstbr** [comment]
Closing in favor of https://github.com/styled-components/stylelint-processor-styled-components/issues/128, would appreciate you adding a real world test to the processor repo!
